backend/route/contributors.route.js

```diff
@@ -0,0 +1,206 @@
+// const Boom = require('@hapi/boom');
+// const Joi = require('@hapi/joi');
+// const { getExperience } = require('../service/experience.service');
+// const { Developer, Attendance } = require('../models');
+// const developerValidator = require('../validators/developer.validator');
+// const { update: validateAttendanceUpdate } = require('../validators/attendance.validators');
+// const { email: emailValidator, id: idValidator } = require('../validators/value.validators');
+const axios = require('axios');
+
+// const getUser = (email, shouldExplode = true) => Developer.findOne({ where: { email } })
+//     .then((developer) => {
+//         if (!developer && shouldExplode) {
+//             throw Boom.notFound('Could not find developer with that email.');
+//         } else {
+//             return developer;
+//         }
+//     });
+
+// const getAttendance = (DeveloperId, ExperienceId, shouldExplode = true) => Attendance.findOne({ where: { DeveloperId, ExperienceId } })
+//     .then((attendance) => {
+//         if (!attendance && shouldExplode) {
+//             throw Boom.notFound('Could not find Attendance with that DeveloperId and ExperienceId');
+//         } else {
+//             return attendance;
+//         }
+//     });
+
+const getLinkedInUrl = username => {
+    const linkedIn = {
+        'sophyphreak': 'https://www.linkedin.com/in/sophyphreak/'
+    }
+    return linkedIn[username] || null
+}
```

Comment:
Nice. This will do for now. Once auth is in place, we'll want to move to a database-driven solution.
Can you please move the `const linkedIn` to a separate file called `contributors.js` so that we can ask contributors to upload their profile content directly inside that single file?

Reply:
Can do.